src/ontology/mondo-edit.obo

```diff
@@ -438204,6 +438184,44 @@ is_a: MONDO:0100249 {source="OMIM:400045"} ! 46,XX testicular disorder of sex de
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/11311 {source="OMIM:400045", source="mim2gene_medgen"} ! SRY
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100253
+name: vertigo, benign recurrent, 1
+synonym: "vertigo, benign recurrent, 1" EXACT [OMIM:193007]
+xref: OMIM:193007 {source="MONDO:directSiblingOf"}
+is_a: MONDO:0003847 {source="OMIM:193007", source="https://github.com/monarch-initiative/mondo/issues/962", source="https://orcid.org/0000-0001-5208-3432"} ! Mendelian disease
+is_a: MONDO:0004900 {source="OMIM:193007", source="https://github.com/monarch-initiative/mondo/issues/962", source="https://orcid.org/0000-0001-5208-3432"} ! peripheral vertigo
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100254
+name: benign paroxysmal positional nystagmus
+synonym: "benign paroxysmal positional nystagmus" EXACT [MONDO:0008656]
+synonym: "benign paroxysmal positional vertigo" EXACT [DOID:13941, ICD9CM_2006:386.11]
+synonym: "BPPV" RELATED ABBREVIATION [GARD:0005915]
+synonym: "BRV" RELATED ABBREVIATION [MONDO:Lexical, OMIM:193007]
+synonym: "familial benign recurrent vertigo" RELATED [GARD:0005915]
+synonym: "familial vestibulopathy" RELATED [GARD:0005915]
+synonym: "vertigo, benign paroxysmal positional" RELATED [OMIM:193007]
+synonym: "vertigo, benign recurrent" RELATED [MONDO:Lexical, OMIM:193007]
+synonym: "vertigo, benign recurrent; BRV" RELATED [OMIM:193007]
+synonym: "vestibulopathy, familial" RELATED [OMIM:193007]
+xref: COHD:81878 {source="MONDO:equivalentTo"}
+xref: DOID:13941 {source="MONDO:equivalentTo"}
+xref: GARD:0005915 {source="MONDO:equivalentTo", source="OMIM-shared"}
+xref: ICD9:386.11 {source="MONDO:equivalentTo", source="i2s", source="DOID:13941"}
+xref: MESH:D065635 {source="DOID:13941"}
+xref: OMIM:193007 {source="MONDO:equivalentTo", source="DOID:13941"}
```

Comment:
the OMIM entry does not mention nystagmus

Reply:
The MESH (MESH:D065635) definition is:
Idiopathic recurrent VERTIGO associated with POSITIONAL NYSTAGMUS. It is associated with a vestibular loss without other neurological or auditory signs. Unlike in LABYRINTHITIS and VESTIBULAR NEURONITIS, inflammation in the ear is not observed.
(and this is what was used on the Mondo record).

I think this label came from DO (https://www.ebi.ac.uk/ols/ontologies/doid/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FDOID_13941).

Should we:
1) have a term labeled 'benign paroxysmal positional nystagmus' with this def that xrefs MESH (and UMLS:C0155502, as they use the same def)
2) have a separate term labeled VERTIGO, BENIGN RECURRENT that xrefs [OMIM](https://omim.org/entry/193007) and [GARD](https://rarediseases.info.nih.gov/diseases/5915/benign-paroxysmal-positional-vertigo)?
3) should VERTIGO, BENIGN RECURRENT, 1 be a it's own Mondo class or should it be a synonym on the Mondo term that is equiv to OMIM:193007 VERTIGO, BENIGN RECURRENT (note OMIM has both of these terms on the same record).